From 572fff7b8ecacb841e0fc4620c5d826f4c1a64ca Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Thu, 29 Apr 2021 08:15:52 -0500
Subject: [PATCH 82/83] arm64: dts: beacon-imx8mn-som: Increase Bluetooth UART
 to 3MBps

In order to run the UART > 1.5MBps, the UART needed to be reparented to
a clock faster than 24MHz. With that completed, the UART speed can be
increased to 3MBps.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/beacon-imx8mn-som.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mn-som.dtsi b/arch/arm64/boot/dts/freescale/beacon-imx8mn-som.dtsi
index 5ad50c1adcaa..ad168844ddeb 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mn-som.dtsi
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mn-som.dtsi
@@ -223,7 +223,7 @@
 		shutdown-gpios = <&gpio2 6 GPIO_ACTIVE_HIGH>;
 		clocks = <&osc_32k>;
 		clock-names = "extclk";
-		max-speed = <1000000>;
+		max-speed = <3000000>;
 	};
 };
 
-- 
2.17.1

